### ISSUE TITLE
ANDROID: aidl compiler crash with the new Android SDK Build-Tools 29

### ISSUE THREAD

**Issue by RonnySchleicher** [j3k0/cordova-plugin-purchase#884]
### system info
* Windows 10 Pro 64 Bit
* Android Studio
![image](https://user-images.githubusercontent.com/1914203/60387155-754d7980-9a9f-11e9-95c1-075e6dacf71e.png)
![image](https://user-images.githubusercontent.com/1914203/60387174-a037cd80-9a9f-11e9-9c5c-e5fa9626109c.png)
![image](https://user-images.githubusercontent.com/1914203/60387184-b2197080-9a9f-11e9-9fa8-67fb43527907.png)

* cordova -v
  * cordova 9.0.0 (cordova-lib@9.0.1)
* cordova plugin ls
  * cc.fovea.cordova.purchase 8.1.1 "Purchase"
  * cordova-plugin-whitelist 1.3.3 "Whitelist"

### Expected behavior
* compile/generate the project 😄 

### Observed behavior
* don't compile/generate the project 😢 

### Steps to reproduce
* Install/Update the latest version of the Android Studio AND the new **Android SDK Build-Tools 29** ❗️ 
* Execute follow little **Batch** to generate a simple Demo App with the cordova-plugin-purchase
```
call npm uninstall -g cordova
call npm install -g cordova
call cordova create Cordova_android de.test.test Test
cd Cordova_android
call cordova platform add android
call cordova plugin add cc.fovea.cordova.purchase --variable BILLING_KEY="xyz... your own key"
```

* Now import the Android Project in your Android Studio.
* After that you have to remove the min SDK entries from the two Manifest files. But this is a Cordova Bug and not important for this issue.
* When you now make a Build your become follow error message(s)
```
Process 'command 'C:\Users\ronny\AppData\Local\Android\Sdk\build-tools\29.0.0\aidl.exe'' finished with non-zero exit value -1073741819
```
![image](https://user-images.githubusercontent.com/1914203/60387365-07ef1800-9aa2-11e9-843a-fb7d5028bd54.png)

```
Caused by: org.gradle.process.internal.ExecException: Process 'command 'C:\Users\ronny\AppData\Local\Android\Sdk\build-tools\29.0.0\aidl.exe'' finished with non-zero exit value -1073741819
	at org.gradle.process.internal.DefaultExecHandle$ExecResultImpl.assertNormalExitValue(DefaultExecHandle.java:396)
	at com.android.build.gradle.internal.process.GradleProcessResult.assertNormalExitValue(GradleProcessResult.java:46)
	... 12 more
```
![image](https://user-images.githubusercontent.com/1914203/60387379-60261a00-9aa2-11e9-8fbd-a9f18384fb83.png)

## ❗️ Important Note ❗️ 
* I hav testet the problem on two different PCs with the same bad result.
* With the 'old' build-tools 28.0.3 everthing works fine ❗️

**Comment by RonnySchleicher** [comment]
## ❗️Second Important Note ❗️ 
* On the iMac works fine with the build-tools 29.0.0
* The Cordova Setup are the same like the Windows PC above
* That's the imac I tested with
![image](https://user-images.githubusercontent.com/1914203/60432002-6ed71300-9c01-11e9-8762-82e745b025e9.png)

**Comment by RonnySchleicher** [comment]
https://stackoverflow.com/questions/56510253/android-aidl-error-sdk-build-tools-29-0-0#new-answer

**Comment by RonnySchleicher** [comment]
https://issuetracker.google.com/issues/132344897

**Comment by RonnySchleicher** [comment]
Hello. 
Is there any news about this issue?

**Comment by marcelinojung** [comment]
today, build-tool 29.0.1 is released and it's fixed!

**Comment by RonnySchleicher** [comment]
Fixed with Build-tool 29.0.1!

**Comment by smager** [comment]
Mine is solved.  

Delete all files under this folder: 

C:\Users\ <username> \AppData\Local\Android\Sdk\build-tools

VSCode or Android studio will automatically regenerate the files on your local machine during pre-compilation.
